src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -733,6 +734,7 @@ namespace OpenRCT2::Ui::Windows
                 }
 
                 gDropdownItems[numPathTypes].Format = kStringIdNone;
+                gDropdownTooltips[numPathTypes] = kStringIdNone;
```

Comment:
Any reason why you don’t show a tooltip for legacy footpaths? It seems a bit inconsistent.

Reply:
I couldn't find a way to get their names as string ids, only as std::strings. If you have a suggestion, I'm all ears. It is inconsistent this way, indeed.

Reply:
You can call `GetLegacyData()`, which contains a field called `string_idx`.

Reply:
Thanks! I see that's already used to assign `pathEntry` in the same function, so that's perfect then.